lib/classes/CLI.js

```diff
@@ -220,6 +220,39 @@ class CLI {
     console.log('');
   }
 
+  printError(e) {
+    this.asciiGreeting();
```

Comment:
I removed it from the error stuff, instead I'm greeting on serverless init. You're right, greeting shouldn't have anything to do with errors, specially that errors are thrown mid-way when another greeting has already fired